tests/scripts/ci.py

```diff
@@ -221,6 +317,10 @@ def main():
         add_subparser(func, subparsers)
 
     args = parser.parse_args()
+    if args.command is None:
```

Comment:
possible to use required=True?

Reply:
I think the help message with that is worse than just printing out the normal help:

```
usage: ci.py [-h] {docs,serve-docs,lint} ...                                    
ci.py: error: the following arguments are required: command                     
```

vs

```
usage: ci.py [-h] {docs,serve-docs,lint} ...

Run CI scripts locally via Docker

positional arguments:
  {docs,serve-docs,lint}
    docs                Build the documentation from gallery/ and docs/. By
                        default this builds only the Python docs. If you are
                        on a CPU machine, you can skip the tutorials and build
                        the docs with the '--precheck --cpu' options.
    serve-docs          Serve the docs using Python's http server
    lint                Run CI's Sanity Check step

optional arguments:
  -h, --help            show this help message and exit
```

Reply:
ah yup seems reasonable